<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tousLivres</class>
 <widget class="QDialog" name="tousLivres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1581</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/logo.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Tous les livres</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_livre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>1541</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="columnCount">
    <number>12</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID LIVRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TITRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DESCRIPTION</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LANGUE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATE PULICATION</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CATEGORIE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AUTEUR</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PRIX</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID RAYON</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID ETAGER</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NOMBRE PAGES</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bt_recherche">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>150</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#bt_recherche{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_recherche:hover{

background-color:#27a9e3;

}</string>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reload">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>80</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#bt_reload{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_reload:hover{

background-color:#27a9e3;

}</string>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criteres de recherche :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ID LIVRE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TITRE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DESCRIPTION</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LANGUE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DATE PUBLICATION</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CATEGORIE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AUTEUR</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PRIX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID RAYON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID ETAGER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NOMBRE PAGES</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ISBN</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="tb_search">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
